sdk/ml/azure-ai-ml/tests/dataset/unittests/test_data_operations.py

```diff
@@ -129,6 +129,7 @@ def test_create_with_spec_file(
                 sas_uri=None,
                 artifact_type=ErrorTarget.DATA,
                 show_progress=True,
+                ignore_file=None,
```

Comment:
default value is None so we can skip this change?

Reply:
I haven't checked this in detail, but if skip this change, similar tests will break
will check this today deep

Reply:
`assert_called_once_with` will not pass when ignoring `ignore_file`, so maybe we shall keep this?

Reply:
this is weird for me. Let me do some further investigation

Reply:
Yes, we should keep this